vsm/lib/raster/esa_s2.cpp

```diff
@@ -378,9 +378,9 @@ bool ESA_S2_Image::splitJP2(const std::filesystem::path &path_in, const std::fil
 
 			std::filesystem::create_directories(ss_path_out.str());
 
-			ss_path_out_nc << ss_path_out.str() << "bands.nc";
-			ss_path_out_png << ss_path_out.str() << path_in.stem().string() << "_" << tx0 << "_" << ty0 << ".png";
-
+			ss_path_out_nc << ss_path_out.str() << path_in.stem().string().substr(0, path_in.stem().string().find_last_of("_")) << "_tile_" << xi << "_" << yi << ".nc";
```

Comment:
Perhaps we could have a function which performs
`path_in.stem().string().substr(0, path_in.stem().string().find_last_of("_"))`